### PR TITLE
Add overrides for GitLab resource limits

### PR DESCRIPTION
**Pull request by jacobbmay**
- Added overrides for GitLab resource settings
- Configured default settings to align closer with GitLab 2k user architecture

### CODE REVIEW

**Review by anthonywendt — DISMISSED** [review]
LGTM

File: uds-config/uds-core-swf/test-cluster/uds-config.yaml [hunk 1]
```diff
@@ -41,6 +41,40 @@ variables:
     GITLAB_BACKUP_EXTRA_ARGS: "--skip artifiacts,registry"
     BUCKET_SUFFIX: "-test"
     GITLAB_REDIS_ENDPOINT: "redis-master.dev-redis.svc.cluster.local"
+    # # Overrides for scaled down cluster for local dev and CI
```
Comment (blancharda) [inline uds-config/uds-core-swf/test-cluster/uds-config.yaml:44]: Same.. 
```suggestion
```

**Review by blancharda — CHANGES_REQUESTED** [review]
Looks good -- super minor changes to clean up some comments

**Review by blancharda — APPROVED** [review]
LGTM

**Review by anthonywendt — APPROVED** [review]

**Pull request merged**